src/snowflake/snowpark/modin/plugin/docstrings/base.py

```diff
@@ -443,6 +443,60 @@ def aggregate():
     def align():
         """
         Align two objects on their axes with the specified join method.
+
```

Comment:
We should add notes here that explain what isn't supported, e.g. axis= 1 or None, etc.